examples/tutorials/nb_python/Habitat_Lab.py

```diff
@@ -418,7 +418,7 @@ def act(self, observations):
     pass
 
 # %% [markdown]
-# Deploy habitat-sim trained models on real robots with the [habitat-pyrobot bridge](https://github.com/facebookresearch/habitat-lab/blob/71d409ab214a7814a9bd9b7e44fd25f57a0443ba/habitat/sims/pyrobot/pyrobot.py)
+# Deploy habitat-sim trained models on real robots with the [habitat-pyrobot bridge](https://github.com/facebookresearch/habitat-lab/blob/main/habitat-lab/habitat/sims/pyrobot/pyrobot.py)
```

Comment:
I think we should keep a specific commit here (Does not need to be the same commit number) but this would help us make sure changes to main will not invalidate what the notebooks are saying.